model/fn-execution/build.gradle

```diff
@@ -28,6 +28,39 @@ ext.summary = "Portable definitions for execution user-defined functions."
 dependencies {
   // We purposely depend on the unshaded classes for protobuf compilation and
   // export the shaded variant as the actual runtime dependency.
+  compile "com.google.guava:guava:28.1-android"
   compile project(path: ":model:pipeline", configuration: "unshaded")
   runtime project(path: ":model:pipeline", configuration: "shadow")
+  permitUnusedDeclared library.java.vendored_grpc_1_26_0
+  permitUnusedDeclared library.java.error_prone_annotations
+  permitUnusedDeclared "com.github.jponge:lzma-java:1.3"
```

Comment:
unusedDeclaredArtifacts: 
   - com.github.jponge:lzma-java:1.3@jar
   - com.google.api.grpc:proto-google-common-protos:1.12.0@jar
   - com.google.auth:google-auth-library-credentials:0.18.0@jar
   - com.google.code.gson:gson:2.8.6@jar
   - com.google.errorprone:error_prone_annotations:2.3.3@jar
   - com.google.guava:guava:28.1-android@jar
   - com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-5@jar
   - com.google.protobuf:protobuf-java-util:3.11.0@jar
   - com.jcraft:jzlib:1.1.3@jar
   - com.ning:compress-lzf:1.0.3@jar
   - commons-logging:commons-logging:1.2@jar
   - io.grpc:grpc-auth:1.26.0@jar
   - io.grpc:grpc-context:1.26.0@jar
   - io.grpc:grpc-core:1.26.0@jar
   - io.grpc:grpc-netty:1.26.0@jar
   - io.grpc:grpc-testing:1.26.0@jar
   - io.netty:netty-tcnative-boringssl-static:2.0.33.Final@jar
   - io.netty:netty-transport-native-epoll:4.1.51.Final@jar
   - io.opencensus:opencensus-api:0.24.0@jar
   - io.opencensus:opencensus-contrib-grpc-metrics:0.24.0@jar
   - io.perfmark:perfmark-api:0.19.0@jar
   - net.jpountz.lz4:lz4:1.3.0@jar
   - org.apache.beam:beam-vendor-grpc-1_26_0:0.3@jar
   - org.apache.logging.log4j:log4j-api:2.6.2@jar
   - org.bouncycastle:bcpkix-jdk15on:1.54@jar
   - org.bouncycastle:bcprov-jdk15on:1.54@jar
   - org.conscrypt:conscrypt-openjdk-uber:2.5.1@jar
   - org.eclipse.jetty.alpn:alpn-api:1.1.2.v20150522@jar
   - org.eclipse.jetty.npn:npn-api:1.1.1.v20141010@jar
   - org.jboss.marshalling:jboss-marshalling:1.4.11.Final@jar
   - org.jboss.modules:jboss-modules:1.1.0.Beta1@jar
   - org.slf4j:slf4j-api:1.7.30@jar

This list of dependencies is transitively included from GrpcVendoring_1_26_0.groovy. Instead of removing it from GrpcVendoring_1_26_0.groovy, i permitted unusedDeclaredArtifacts model/fn-execution/build.gradle.